src/types/User.ts

```diff
@@ -11,4 +11,6 @@ export type User = {
 export interface RequestWithUser extends Request {
     user?: User;
     isAuthenticated(): boolean;
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    query: any;
```

Comment:
To leverage the full usage of typescript we want to make sure to avoid setting items to implicit at much as possible.

 ⚙️ It's best practice to run everything on strict especially when you're first getting used to the language. 

```suggestion
    query?: string | 'whatever else it could be';
```